Packs/Reco/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Reco",
     "description": "Reco is the leader in Dynamic SaaS Security — the only approach that eliminates the SaaS Security Gap (the growing gap between what you can protect and what’s outpacing your security).",
     "support": "partner",
-    "currentVersion": "1.7.0",
+    "currentVersion": "1.8.0",
```

Comment:
it's not a change that requires a minor version upgrade, so 1.7.1 should be good.